redis/exceptions.py

```diff
@@ -29,18 +29,23 @@ class TimeoutError(RedisError):
 
 
 class BusyLoadingError(ConnectionError):
+    "Raised when the Redis server is starting up and not accepting commands yet"
```

Comment:
This line is 1 character too long. PEP-8 doesn't like it, can you multi-line please?